src/development/platform-integration/ios/apple-frameworks.md

```diff
@@ -0,0 +1,200 @@
+---
```

Comment:
**suggestion:** If this is a series of comparisons, use subheads for each language, as in:
```markdown
#### Apple Framework
...
#### Flutter
...
```

This would read more smoothly than `In native code, do...` then `In Flutter, do...`

Reply:
Will hold off on this in case we turn it into a table